modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/util/BaseQueryContext.java

```diff
@@ -132,7 +133,7 @@ public List<MetadataHandler<?>> handlers(Class<? extends MetadataHandler<?>> hnd
         );
 
         cluster.setMetadataQuerySupplier(() -> {
-            throw new AssertionError(cantBeUsedMsg);
+            return RelMetadataQueryEx.create();
```

Comment:
@korlov42 Thanks. Should have been removed. This is code is used to debug tests in case of serialisation mismatches (because when it happens you get this error instead of a mismatch).

Reply:
Fixed.